content/actions/reference/events-that-trigger-workflows.md

```diff
@@ -41,6 +41,12 @@ The following steps occur to trigger a workflow run:
 
 The `schedule` event allows you to trigger a workflow at a scheduled time.
 
+{% note %}
+
+Note: Currently due to load `schedule` actions can have up to an hour delay, this is being looked into but please make considerations for this in the interim.
```

Comment:
@BrentonWheeler Thank you for your patience while I discussed this with the engineers! Here is a suggested change after discussing with them:

```suggestion
Note: Due to load, the `schedule` event may be delayed 
```

Reply:
Since you and others have been waiting for this update, I'm going to go ahead and incorporate the suggested change and get this merged down for you.